virttest/utils_test/libvirt.py

```diff
@@ -1594,21 +1594,27 @@ def cleanup_dest_vm(self, vm, srcuri, desturi):
         # Set connect uri back to local uri
         vm.connect_uri = srcuri
 
-    def check_vm_state(self, vm_name, state='paused', uri=None):
+    def check_vm_state(self, vm_name, state='paused', reason=None, uri=None):
```

Comment:
This would break virsh_migate.py and virsh_migration.py,

It is better to always add new arguments at last for better backward compatibility or to change all the users accordingly.

Reply:
@balamuruhans  only virsh_cmd/domain/virsh_migrate_stress.py is using this method on latest code. Updated in tp-libvirt 2225, please help review.

Reply:
Thank you @Yingshun, for taking care of it.